app/content/react/guides/widgets.md

```diff
@@ -130,3 +134,35 @@ Requires/accepts the following additional properties:
 | `xAxisFormatter`   | Function to format X axis values.  |
 | `yAxisFormatter`   | Function to format X axis values. |
 | `tooltipFormatter` | Function to format the tooltip values. |
+
+### Time Series widget
+
+Groups features into time intervals and allows to play an animation that filters the features displayed based on the current interval. 
+
+![TimeSeries Widget](/img/react/time-series-widget.png)
```

Comment:
I would suggest being consistent, and using either TimeSeries or Time Series naming (I would prefer the 1st, but it's up to you). Same for ScatterPlot